src/index.css

```diff
@@ -1,27 +1,422 @@
+.btn {
+  text-transform: uppercase;
+  background: var(--clr-primary);
+  color: var(--clr-white);
+  padding: 0.375rem 0.75rem;
+  letter-spacing: var(--spacing);
+  display: inline-block;
+  font-weight: 700;
+  transition: var(--transition);
+  font-size: 0.875rem;
+  border: none;
+  cursor: pointer;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+}
+
+.btn:hover {
+  color: var(--clr-primary);
+  background: var(--clr-primary-light);
+}
+
+/*
+===============
+Navbar
+===============
+*/
+.loading {
+  text-align: center;
+  margin-top: 5rem;
+}
+
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-bottom: 1px solid gray;
+  border-bottom: #e5e5e5;
+  box-shadow: 0 8px 16px rgb(64 83 252 / 24%);
+  padding: 5px;
+}
+
+.nav-container h1 {
+  width: auto;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  margin-left: 30px;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav-container ul {
+  display: flex;
+  flex-direction: row;
+  margin-left: 40px;
+  text-decoration: none;
+}
+
+.nav-container ul li {
+  list-style: none;
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
+}
+
+.Oval-2 {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+  border-radius: 50px;
+  border-image-source: linear-gradient(to bottom, #307bbe, #379cf6);
+  border-image-slice: 1;
+}
+
+.Line-2 {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.Mask {
+  width: auto;
+  height: 1.063rem;
+}
+
+.Oval {
+  width: 2.813rem;
+  height: 2.813rem;
+  margin: 0 0 0 41.5rem;
+  padding: 0.875rem;
+  border: solid 1px #e8e8e8;
+  border-radius: 50px;
+}
+
+.panel-bg {
+  /* width: 87.5rem; */
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--white-two);
+}
+
+.Lesson-Panel {
+  width: fit-content;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+}
+
+.School-of {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.Title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
```

Comment:
- [ ] The issue where your book component overlaps its container has not been fixed, I noticed that you have very high values for the width of this book title, please remove it completely and use a flex property `flex-grow`. This will allow your app to respond well irrespective of the screen size.

See your work now 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/232166410-c15062f2-840b-44d6-8f7e-21a3e695b3a0.png)

You probably are not having this issue at your end because you are using a wider screen. As a good developer, your website should be responsive to all screen sizes. So using fixed width works against this aim. Kindly fix this and try to resize your app to see how they appear on slightly smaller screens. 

After I remove the fixed width of `12.313rem` on the book `.title` and also reduced the ` margin: 0.188rem 20.188rem 0 0;` margin right from `20.188rem` to `1rem`, I got a better display as shown below 👇🏾 . Please notice the logo is even breaking down and overflowing. Please fix this. 
![image](https://user-images.githubusercontent.com/84156714/232166891-7d3fefd2-fb92-4895-a128-fdc2e74f5fd5.png)